indico/modules/events/editing/client/js/components/FileTypeManager.jsx

```diff
@@ -41,6 +42,17 @@ function fileTypesReducer(state, action) {
   }
 }
 
+function StatusIcon({fileParam, iconName, text}) {
+  return (
+    <Icon
+      size="small"
+      color={fileParam ? 'blue' : 'grey'}
+      name={iconName}
+      title={fileParam ? text : ''}
```

Comment:
You can just use `text` (defaultProp for `text` is null which is valid here)

Reply:
There shouldn't be a default (as the prop is always specified); and `title={false}` would be very odd (and likely a proptypes violation)

Reply:
I meant that `title` has default `null` which works fine in this case

Reply:
ahh, I see now.. nvm my comment